util/dvsim/FlowCfg.py

```diff
@@ -60,6 +60,9 @@ def __init__(self, flow_cfg_file, proj_root, args):
         # a special key 'use_cfgs' within the hjson cfg.
         self.is_master_cfg = False
 
+        # Generated data structures
+        self.deploy = []
```

Comment:
I think it would be better to add comment explaining how the `deploy` variable works. 
For a master cfg, it is the aggregated list of all deploy objects under self.cfgs
For a non-master cfg, its the list of items slated for dispatch.

Reply:
Yes, that's what I meant in my first comment. Suggested adding comment explaining how `deploy` works.

Reply:
Added comment.